docs/pppoe.md

```diff
@@ -64,7 +87,7 @@ uci set network.wan.proto='pppoe'
 uci set network.wan.username='test'
 uci set network.wan.password='password'
 uci set network.wan.ipv6=0
-uci set network.wan.keepalive="1000 5"
+uci set network.wan.keepalive="100 5"
```

Comment:
The default value is "5 1"

Reply:
Yes, but it fails because it's too quick (at least for the PPPoE test srver)